instrumentation/httpclient5/src/test/java/brave/httpclient5/ITTracingCachingH2ClientBuilder.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.httpclient5;
+
+import brave.handler.MutableSpan;
+import brave.propagation.CurrentTraceContext.Scope;
+import brave.propagation.SamplingFlags;
+import brave.propagation.TraceContext;
+import java.io.IOException;
+import java.util.Arrays;
+import okhttp3.mockwebserver.MockResponse;
+import org.apache.hc.client5.http.impl.async.CloseableHttpAsyncClient;
+import org.apache.hc.client5.http.impl.cache.CachingH2AsyncClientBuilder;
+import org.junit.Test;
+
+import static brave.Span.Kind.CLIENT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ITTracingCachingH2ClientBuilder extends ITTracingH2AsyncClientBuilder {
```

Comment:
```suggestion
class ITTracingCachingH2ClientBuilder extends ITTracingH2AsyncClientBuilder {
```

Reply:
This is a test code, so I should keep it as public?

Reply:
sorry yeah we are still junit 4. my mistake